packages/react-instantsearch-hooks-dom/src/widgets/__tests__/InfiniteHits.test.tsx

```diff
@@ -0,0 +1,214 @@
+import { act, fireEvent, render, waitFor } from '@testing-library/react';
+import React from 'react';
+
+import { createSearchClient } from '../../../../../test/mock';
+import {
+  createMultiSearchResponse,
+  createSingleSearchResponse,
+} from '../../../../../test/mock/createAPIResponse';
+import { InstantSearchHooksTestWrapper, wait } from '../../../../../test/utils';
+import { InfiniteHits } from '../InfiniteHits';
+
+type CustomHit = {
+  somethingSpecial: string;
+};
+
+const searchClient = createSearchClient({
+  search: (requests) =>
+    Promise.resolve(
+      createMultiSearchResponse(
+        ...requests.map((request) => {
+          const { hitsPerPage = 3, page = 0 } = request.params!;
+          const hits = Array.from({ length: hitsPerPage }, (_, i) => {
+            const offset = hitsPerPage * page;
+            return {
+              objectID: (i + offset).toString(),
+              somethingSpecial: String.fromCharCode(
+                'a'.charCodeAt(0) + i + offset
+              ),
+            };
+          });
+
+          return createSingleSearchResponse<CustomHit>({
+            hits,
+            page,
+            nbPages: 10,
+            hitsPerPage,
+            index: request.indexName,
+          });
+        })
+      )
+    ),
+});
+
+describe('InfiniteHits', () => {
+  test('renders with default props', async () => {
+    const { container } = render(
+      <InstantSearchHooksTestWrapper searchClient={searchClient}>
+        <InfiniteHits />
+      </InstantSearchHooksTestWrapper>
+    );
+
+    await waitFor(() => {
+      expect(container.querySelector('.ais-InfiniteHits'))
+        .toMatchInlineSnapshot(`
+        <div
+          class="ais-InfiniteHits"
+        >
+          <ol
+            class="ais-InfiniteHits-list"
+          />
```

Comment:
I'm wondering if there should be a list of three items using the default hit component here?